core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -158,8 +158,22 @@ object ConsumerPerformance {
     val elapsedMs: Double = endMs - startMs
     val totalMBRead = (bytesRead * 1.0) / (1024 * 1024)
     val mbRead = ((bytesRead - lastBytesRead) * 1.0) / (1024 * 1024)
+    
```

Comment:
@vahidhashemian I agree with you that currently `reporting-interval` may be 0 or even negative. But does such value even make sense for reporting-interval? In my opinion, we should either throw exception, or disable printMessage until the end of experiment, if `reporting-interval <=0`. What do you think?